### PR TITLE
Styled bottom nav, icons.

### PR DESCRIPTION
**Pull request by DjordjeVujatovic**

### CODE REVIEW

File: js/navigation/NavigationLayout.ios.js [hunk 1]
```diff
@@ -10,31 +10,22 @@ import SignInSignUpScreen from '../scenes/SignInSignUpScreen'
 import AccountScreen from '../scenes/AccountScreen'
 import MyPotLucksScreen from '../scenes/MyPotlucksScreen'
 import { hostTabScreens, hostTabStyles, guestTabScreens, guestTabStyles } from './TabNavConfig'
-
+import { bottomNavigatorScreens, bottomNavigatorStyles } from './BottomNavConfig'
```
Comment (BlakeTurner) [inline js/navigation/NavigationLayout.ios.js:13]: Nice work.

File: js/scenes/AccountScreen/AccountScreenContainer.js [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 import AccountScreen from './AccountScreen'
 
 class AccountScreenContainer extends React.Component {
   static navigationOptions = {
-    title: 'Account Screen',
+    title: 'Account',
+    
+    tabBar:()=> ({
+      icon: (
+        <Image style={{height:30, width: 25}} source={require('../../../assets/images/accounticon.png')} />
```
Comment (BlakeTurner) [inline js/scenes/AccountScreen/AccountScreenContainer.js]: Is there a reason this is the only one with a height of 30?

File: js/navigation/BottomNavConfig.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      fontSize: typography.smallestFontSize,
+      color: 'white',
+      paddingBottom: 8
+    },
+    style: {
+      backgroundColor: colors.mainBrandColor,
+      height: windowHeight*.09,
+    }
+    }
+  }
```
Comment (BlakeTurner) [inline js/navigation/BottomNavConfig.js]: Watch indentation at bottom of file

File: js/scenes/AccountScreen/AccountScreenContainer.js [hunk 4]
```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import { Image } from 'react-native';
 import AccountScreen from './AccountScreen'
 
 class AccountScreenContainer extends React.Component {
   static navigationOptions = {
-    title: 'Account Screen',
+    title: 'Account',
+    
+    tabBar:()=> ({
```
Comment (BlakeTurner) [inline js/scenes/AccountScreen/AccountScreenContainer.js]: Cleaner spacing here:

```js
tabBar: () => ({

})
```

File: js/scenes/CreatePotluckScreen/CreatePotluckScreenContainer.js [hunk 5]
```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import { Image } from 'react-native';
 import CreatePotluckScreen from './CreatePotluckScreen'
 
 class CreatePotluckScreenContainer extends React.Component {
   static navigationOptions = {
     title: 'Create Potlucks',
+
+    tabBar:()=> ({
```
Comment (BlakeTurner) [inline js/scenes/CreatePotluckScreen/CreatePotluckScreenContainer.js]: Spacing

File: js/scenes/MyPotlucksScreen/MyPotlucksScreenContainer.js [hunk 6]
```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import { Image } from 'react-native';
 import MyPotlucksScreen from './MyPotlucksScreen'
 
 class MyPotlucksScreenContainer extends React.Component {
   static navigationOptions = {
-    title: 'MyPotlucks',
+    title: 'My Potlucks',
+
+    tabBar:()=> ({
```
Comment (BlakeTurner) [inline js/scenes/MyPotlucksScreen/MyPotlucksScreenContainer.js]: Spacing

**Review by BlakeTurner — CHANGES_REQUESTED** [review]

File: js/scenes/AccountScreen/AccountScreenContainer.js (hunk 2 above)
Comment (DjordjeVujatovic) [reply js/scenes/AccountScreen/AccountScreenContainer.js]: Just for sizing purposes, I changed it around a bit, but the width on this needs to stay at 25.

**Review by BlakeTurner — APPROVED** [review]

**Pull request merged**